check-cla/README.md

```diff
@@ -1,43 +1,55 @@
 # Check CLA (Contributor License Agreement)
 
-This is a custom GitHub action to be used in the conda GitHub organization
-for checking the conda contributor license agreement.
+A custom GitHub action to be used in the conda GitHub organization for checking the
+conda contributor license agreement.
 
 ## GitHub Action Usage
 
 In your GitHub repository include the action in your workflows:
 
 ```yaml
-name: Contributor license agreement (CLA)
+name: Check CLA
 
 on:
   issue_comment:
-    types:
-      - created
+    types: [created]
   pull_request_target:
-    types:
-      - reopened
-      - opened
-      - synchronize
```

Comment:
Why remove these events?

Reply:
these are the default types for this event: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target
<img width="750" alt="Screenshot 2023-04-27 at 2 15 48 PM" src="https://user-images.githubusercontent.com/4546435/234968441-0f9aa96e-e8ae-4cc1-976a-654690dbac85.png">